packages/iota/src/validation/parents/parents.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+import type { HexEncodedString } from "../../models/hexEncodedTypes";
+import { IValidationResult, failValidation, mergeValidationResults } from "../result";
+
+/**
+ * The minimum count of parents in block.
+ */
+export const MIN_PARENTS_COUNT: number = 1;
+/**
+ * The maximum count of parents in block.
+ */
+export const MAX_PARENTS_COUNT: number = 8;
+
+/**
+ * Validate parent ids.
+ * @param parents The parentIds to validate.
+ * @returns The validation result.
+ */
+export function validateParents(
```

Comment:
For this also i wouldn't add a new file, just move this function to block.ts where it is used.